frontend/libs/studio-icons/src/react/icons/NumberIcon.tsx

```diff
@@ -5,52 +5,10 @@ import type { IconProps } from '../types';
 export const NumberIcon = (props: IconProps): React.ReactElement => (
   <SvgTemplate viewBox='0 0 36 36' {...props}>
     <path
-      fillRule='evenodd'
-      clipRule='evenodd'
-      d='
-        M 10.4795 29.8148
-        L 13.4646 5.81476
-        L 16.4417 6.18505
-        L 13.4566 30.1851
-        Z
-      '
       fill='currentColor'
-    />
-    <path
       fillRule='evenodd'
       clipRule='evenodd'
-      d='
-        M 19.4355 29.8148
-        L 22.4207 5.81476
-        L 25.3978 6.18505
-        L 22.4126 30.1851
-        Z
-      '
-      fill='currentColor'
-    />
-    <path
-      fillRule='evenodd'
-      clipRule='evenodd'
-      d='
-        M 6.13124 12.4091
-        L 30.0117 12.5915
-        L 29.989 15.5914
-        L 6.10856 15.409
-        Z
-      '
-      fill='currentColor'
-    />
-    <path
-      fillRule='evenodd'
-      clipRule='evenodd'
-      d='
-        M 6.01112 20.4083
-        L 29.8916 20.5906
-        L 29.8689 23.5905
-        L 5.98844 23.4082
-        Z
-      '
-      fill='currentColor'
+      d='M15.5 12a2.5 2.5 0 1 1 5 0 2.5 2.5 0 0 1-5 0m0 12a2.5 2.5 0 1 1 5 0 2.5 2.5 0 0 1-5 0M10 17.5a1.5 1.5 0 0 0 0 3h16a1.5 1.5 0 0 0 0-3z'
```

Comment:
Det skurrer litt for meg at dette ikonet ikke er symmetrisk om den horisontale aksen. Den nederste prikken er mye nærmere streken enn den øverste. Er det meningen?

Aksel har for øvrig et ikon som likner: https://aksel.nav.no/ikoner/Divide
Vår ikonpakke er bare en utvidelse av Aksels, så dette ikonet er allerede tilgjengelig. Hva med å bruke det i stedet?

Reply:
Bra! På generelt grunnlag bør vi ikke gjøre om så mye på ikonene. Litt av poenget med å bruke Navs ikonpakke er å få ting til å se likt ut på tvers av offentlig sektor. Det gjør også at det blir mindre kode å vedlikeholde for oss. Men hvis du mener det er ting som bør endres på, er Nav åpne for bidrag: https://github.com/navikt/aksel/blob/main/%40navikt/aksel-icons/CONTRIBUTING.md